pkg/core/redis/pool.go

```diff
@@ -1,20 +1,33 @@
 package redis
 
 import (
+	"errors"
 	"fmt"
 	"time"
 
+	"github.com/FZambia/sentinel"
 	"github.com/gomodule/redigo/redis"
 )
 
+// When sentinel is enabled, host and port will be ignored
 type Configuration struct {
-	Host     string `envconfig:"HOST"`
-	Port     int    `envconfig:"PORT" default:"6379"`
-	Password string `envconfig:"PASSWORD"`
-	DB       int    `envconfig:"DB" default:"0"`
+	SentinelEnabled bool           `envconfig:"SENTINEL_ENABLED"`
```

Comment:
Put this in `SentinelConfig` and rename it to `Enabled`?

Reply:
sure! updated:)